nectarchain/calibration/container/waveforms.py

```diff
@@ -0,0 +1,350 @@
+                wfs_hg_tmp[pix]=event.r0.tel[0].waveform[0,self.pixels_id[pix]]
+
+            self.wfs_hg[i] = wfs_hg_tmp
+            self.wfs_lg[i] = wfs_lg_tmp
+
+
+
+        #if compute_trigger_patern and np.max(self.trig_pattern) == 0:
+        #    self.compute_trigger_patern()
+
+
+    def write(self,path : str, **kwargs) : 
+        suffix = kwargs.get("suffix","")
+        if suffix != "" : suffix = f"_{suffix}"
+
+        log.info(f"saving in {path}")
+        os.makedirs(path,exist_ok = True)
+
+        hdr = fits.Header()
+        hdr['RUN'] = self.__run_number
+        hdr['NEVENTS'] = self.nevents
+        hdr['NPIXELS'] = self.npixels
+        hdr['NSAMPLES'] = self.nsamples
+        hdr['SUBARRAY'] = self.subarray.name
+
+        self.subarray.to_hdf(f"{Path(path)}/subarray_run{self.run_number}.hdf5",overwrite=kwargs.get('overwrite',False))
+
+
+
+        hdr['COMMENT'] = f"The waveforms containeur for run {self.__run_number} : primary is the pixels id, 2nd HDU : high gain waveforms, 3rd HDU : low gain waveforms, 4th HDU :  event properties and 5th HDU trigger paterns."
+
+        primary_hdu = fits.PrimaryHDU(self.pixels_id,header=hdr)
+
+        wfs_hg_hdu = fits.ImageHDU(self.wfs_hg)
+        wfs_lg_hdu = fits.ImageHDU(self.wfs_lg)
+
+
+        col1 = fits.Column(array = self.event_id, name = "event_id", format = '1J')
+        col2 = fits.Column(array = self.event_type, name = "event_type", format = '1I')
+        col3 = fits.Column(array = self.ucts_timestamp, name = "ucts_timestamp", format = '1K')
+        col4 = fits.Column(array = self.ucts_busy_counter, name = "ucts_busy_counter", format = '1J')
+        col5 = fits.Column(array = self.ucts_event_counter, name = "ucts_event_counter", format = '1J')
+        col6 = fits.Column(array = self.multiplicity, name = "multiplicity", format = '1I')
+
+        coldefs = fits.ColDefs([col1, col2, col3, col4, col5, col6])
+        event_properties = fits.BinTableHDU.from_columns(coldefs)
+
+        col1 = fits.Column(array = self.trig_pattern_all, name = "trig_pattern_all", format = f'{4 * self.CAMERA.n_pixels}L',dim = f'({self.CAMERA.n_pixels},4)')
+        col2 = fits.Column(array = self.trig_pattern, name = "trig_pattern", format = f'{self.CAMERA.n_pixels}L')
+        coldefs = fits.ColDefs([col1, col2])
+        trigger_patern = fits.BinTableHDU.from_columns(coldefs)
+
+        hdul = fits.HDUList([primary_hdu, wfs_hg_hdu, wfs_lg_hdu,event_properties,trigger_patern])
+        try : 
+            hdul.writeto(Path(path)/f"waveforms_run{self.run_number}{suffix}.fits",overwrite=kwargs.get('overwrite',False))
+            log.info(f"run saved in {Path(path)}/waveforms_run{self.run_number}{suffix}.fits")
+        except OSError as e : 
+            log.warning(e)
+        except Exception as e :
+            log.error(e,exc_info = True)
+            raise e
+
+
+
+    @staticmethod
+    def load(path : str) : 
```

Comment:
Document that this method reads two different input files at once, and what they are (or expected to be).